KomaMRICore/test/Project.toml

```diff
@@ -1,6 +1,6 @@
 [deps]
-HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
-KomaMRIFiles = "fcf631a6-1c7e-4e88-9e64-b8888386d9dc"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
```

Comment:
:eyes:

Reply:
Addresed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/e37397a7c4d95cc942a71195de6c52d1aeacfe01